### PR TITLE
Don't override fabric env vars if they are already set.

### PR DESCRIPTION
**Pull request by ashb**
If you invoke fabric with `fab --set aws=foo my_task` the previous way
we were setting these would then _overwrite_ the values you'd set on the
command line with lots of None and leading to hard to diagnose
behaviour.

(It also sets these at import time which could be an extra source of
confusion.)

### CODE REVIEW

File: bootstrap_cfn/fab_tasks.py [hunk 1]
```diff
@@ -13,12 +13,15 @@
 from bootstrap_cfn.utils import tail
 
 
+# Default fab config. Set via the tasks below or --set
+env.setdefault('application')
+env.setdefault('environment')
+env.setdefault('aws')
+env.setdefault('config')
+env.setdefault('password')
```
Comment (ashb) [inline bootstrap_cfn/fab_tasks.py:21]: This should be `stack_passwords` - the env.password was wrong before.

**Comment by pidah** [comment]
env.setdefault('password') should be env.setdefault('passwords')

**Comment by ashb** [comment]
@pidah I think it should be `stack_passwords` as that is what the password task writes to?

**Comment by pidah** [comment]
LGTM

**Pull request merged**